scripts/data_files/driver_templates/psa_crypto_driver_wrappers.c.jinja

```diff
@@ -2332,4 +2332,10 @@ psa_status_t psa_driver_wrapper_mac_abort(
     }
 }
 
```

Comment:
Unit testing: we need to have tests that validate how entropy is counted, and in particular we need negative tests where functions return a failure status or a low entropy estimate. We don't need extensive testing in this pull request, it's ok to advertise the feature as not yet usable for production and to expand the test coverage in follow-ups.

The unit tests will take the form of a test driver that can be told what to do, similar to the existing accelerator and secure element driver tests (`test_suite_psa_crypto_driver_wrappers.function`).